nimbus-features/messagingFeature.yaml

```
@@ -2,6 +2,24 @@ features:
   messaging:
     description: >
       The in-app messaging system
+    meta-bug: ~
+    documentation:
+      - name: User documentation
+        url:  https://experimenter.info/messaging/mobile-messaging
+    contacts:
+      # Nimbus team
+      - jhugman@mozilla.com
+      - brennie@mozilla.com
+      # iOS team
+      - gbuciu@mozilla.com
+      - wtiechmann@mozilla.com
+      - npatel@mozilla.com
```

this person doesn't work at Mozilla anymore, and there's a typo in Winnie's email
```suggestion
      - wteichmann@mozilla.com
```

---

Gosh darn it! Thanks!